index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/73a06aba35.js" crossorigin="anonymous"></script>
+    <title>Portfolio2</title>
+  </head>
+  <body>
+
+
+    <div class="container">
+      <header>
+        <a id="logo" href="index.html">Welcome</a>
+        <nav>
+          <i id="hamburger" class="fa-solid fa-bars"></i>
+        </nav>
+      </header>
+        
+
+      <section class="main-section">
+        <p class="primary-text">
+          <span>Hey there.I'm<br>Hammed<br></span> 
+          <span>I'm a Software Developer</span>
+        </p>
```

Comment:
- [ ] for Semantics & Seo reasons your main title should be an `<h1>` tag instead of `<p>` , kindly fix that and you can read more about it [Here](https://www.reliablesoft.net/h1-tag/) .

Reply:
Done